src/secret.js

```diff
@@ -20,30 +20,39 @@ const secretPath = path.normalize(__dirname + "/../secret");
 let secret;
 
 // Used by the benchmark/loadtest tool.
-const get = function() {
+const get = function(done) {
+  if (secret) {
+    return done(null, done);
```

Comment:
You're right! I missed that in my tests. I'll fix the code and add a test for this.